### PR TITLE
Comment api_key in the default config file

### PR DESCRIPTION
**Pull request by saimn**
Having this key defined breaks the detection of unmodified config file.

Related to (proposed) changes to `astropy.config` (https://github.com/astropy/astropy/pull/10877, https://github.com/astropy/astropy/pull/10893). If this key is not commented, the detection of unmodified config file does not work, unless the config file is strictly the same as the default one.

### CODE REVIEW

**Review by bsipocz — APPROVED** [review]
Looks OK to me, but let's wait the sign-off from the other maintainers before merging.

**Review by keflavich — APPROVED** [review]

**Review by ceb8 — APPROVED** [review]

**Comment by ceb8** [comment]
Thanks @saimn!

**Pull request merged**
